tools/camera/findAdducts.xml

```diff
@@ -0,0 +1,202 @@
+<tool id="CAMERA_findAdducts" name="CAMERA.findAdducts" version="0.1.0+camera@TOOL_VERSION@-galaxy@VERSION_SUFFIX@" profile="23.0">
+
+    <description>Annotate adducts</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="requirements"/>
+
+    <command detect_errors="exit_code"><![CDATA[
```

Comment:
The indentation seems to be off here and makes it hard to read.

I recommend to use the https://github.com/galaxyproject/galaxy-language-server it has an auto-format feature.

Reply:
For me, the indentation seems correct. I validate the lint steps. I've already indented and formatted the code. Is it just the 4 extra spaces that bother you @bgruening ? In all the other tools I've seen, they all have this style of indentation between `<tool></tool>` tags. Maybe I've misunderstood.